### PR TITLE
Drawer navigation

### PR DESCRIPTION
**Pull request by honkoan**
My reflections on how this week's project turned out:

This week was hard. I did what I could. I had lots of problems with my internet connection and with my phone. I don't feel very confident with React Native after this week but I'm really interested to learn more about it. I wish I would have had more time to focus on this week's project. I wondered for too long which focus Area to choose and then I got in a hurry with the project and didn’t have time to finish it.

Things I'd like to have clarified or explained in more detail:

I would have liked to have a little more detailed instructions on what to do and what the requirements are. I could also have asked for help, but I had a busy week and didn’t have time to focus on the project properly until the weekend. So I did what I could.

### CODE REVIEW

File: screens/Home.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const image = {
+  uri:
+    'https://images.unsplash.com/photo-1615224572756-729e7d1cd942?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=934&q=80',
+}
+
+const HomeContainer = styled.View`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vw;
```
Comment (KarolCislowski) [inline screens/Home.js:21]: image

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Unfortunately, I was unable to build your project to run it on my device.

File: screens/Home.js (hunk 1 above)
Comment (honkoan) [reply screens/Home.js]: I changed the height property and now I'm able to run the project on my device.

**Review by KarolCislowski — APPROVED** [review]
Nice 👍🏻

**Pull request closed without merging**